.github/workflows/update-versions.yml

```diff
@@ -0,0 +1,62 @@
+name: Auto Bump Versions
+
+on:
+  issue_comment:                                     
+    types: [created, edited, deleted]
+
+jobs:
+  build:
+    if: ${{ github.event.issue.pull_request }}
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
does it make a difference whether the PR is from a fork or local when this runs? (assume it is fine, but want to make sure the fork PRs work)

Reply:
Nice catch. I seem to have found the mistake I was doing her though let me run a quick tests and push some changes